### PR TITLE
31 design a database structure

### LINKED ISSUE #31: Design a database structure
Points: 5
Design how many tables the database will have, each table’s structure, and the columns within each table.

### PR DESCRIPTION
**Pull request by Fahim-A-Karim**
Basically everything in 29, without the ability to add values to the database

### CODE REVIEW

File: digital_logic_simulator/settings.py [hunk 1]
```diff
@@ -21,10 +21,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = 'a'
 
 # SECURITY WARNING: don't run with debug turned on in production!
```
Comment (joshuas4) [inline digital_logic_simulator/settings.py]: This file should not be changed. Follow instructions as mentioned in readme

**Review by joshuas4 — COMMENTED** [review]
I don't know if the .xml and iml files should be changed, these should probably be ignored.

**Review by joshuas4 — CHANGES_REQUESTED** [review]
This adds in basic gate types and inputs/outputs to the models LogicGate class. These can be used for when logic gates are assigned a type.

Just need to remove settings.py and xml/iml files and it should be good to go.

**Review by Nguyen-Tommy — APPROVED** [review]
Didn't find any issues.

**Pull request merged**